Troubleshooting/Auxiliary-Telescope/Telescope-gets-stuck.rst

```diff
@@ -0,0 +1,27 @@
+######################
+Telescope gets "stuck"
+######################
+
+On a few occasions there have been instances where the telescope appears stuck and does not move (although being enabled and responding to commands).
+This has been rare, and was captured in Jira ticket OBS-53.
+This page provides instructions on how people were able to get it unstuck.
+They are provided for guidance, but should not be considered a strict recipe.
+
+The following list was performed using the EUI (on auxtel-brick.cp.lsst.com), but it may also be possible to do via the CSC.
+This issue happened at AZ: -76 deg, EL: 45 deg. 
+The solution was to command the azimuth a small number of degrees back or forward (exact order is unknown) and the movement reestablished.
+
+
+#. Start and enable pneumatics system (open the Master Air Supply and Instrument Air Valve) even if the status is already marked as open. 
```

Comment:
Could you provide a sequence of screen shots for what needs to happen here?

Reply:
Operators know how to do this I think.